### PR TITLE
DAC6-2601 : Added changes to firstcontact

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -96,12 +96,12 @@ haveTelephone.hint = We will call if we have any questions about your reports.
...
+contactPhone.title = What is the telephone number for the first contact?
 contactPhone.heading = What is the telephone number for {0}?
-contactPhone.hint = Enter a telephone number, like&nbsp;01632&nbsp;960&nbsp;001, 07700&nbsp;900&nbsp;982 or +44&nbsp;808&nbsp;157&nbsp;0192.
-contactPhone.error.required = Enter a telephone number
-contactPhone.error.invalid = Enter a telephone number in the correct format
-contactPhone.error.length = Telephone number must be 24 characters or less
+contactPhone.hint = For international numbers, include the country code.
+contactPhone.error.required = Enter a telephone number for your first contact
+ contactPhone.error.length = The telephone number must be 24 characters or less
+ contactPhone.error.invalid = The telephone number must only include numbers 0 to 9 and a plus sign, like 01632 960 001, 07700 900 982 or +44 808 157 0192
```
Comment (BriWak) [inline conf/messages.en:104]: @AashuChauhanHmrc This is not the right change for DAC6-2601. The change needs to be made to the **Agent** First contact phone page.

**Review by BriWak — CHANGES_REQUESTED** [review]

File: conf/messages.en (hunk 1 above)
Comment (BriWak) [reply conf/messages.en:104]: @AashuChauhanHmrc This is still not reverted back to what it was originally

**Review by BriWak — CHANGES_REQUESTED** [review]

File: conf/messages.en (hunk 1 above)
Comment (AashuChauhanHmrc) [reply conf/messages.en]: reverted changes

Comment (AashuChauhanHmrc) [reply conf/messages.en]: reverted changes

**Review by BriWak — APPROVED** [review]

**Pull request merged**